dm/dumpling/dumpling_test.go

```diff
@@ -124,3 +125,32 @@ func (d *testDumplingSuite) TestDefaultConfig(c *C) {
 	c.Assert(dumpling.dumpConfig.StatementSize, Not(Equals), export.UnspecifiedSize)
 	c.Assert(dumpling.dumpConfig.Rows, Not(Equals), export.UnspecifiedSize)
 }
+
+func (d *testDumplingSuite) TestCallStatus(c *C) {
+	conf := export.DefaultConfig()
+	cfg := &config.SubTaskConfig{}
+	m := &Dumpling{cfg: cfg}
+	ctx := context.Background()
+
+	ss := &binlog.SourceStatus{}
+	s := m.Status(ss).(*pb.DumpStatus)
```

Comment:
we can use `m.Status(nil).(*pb.DumpStatus)` so no need `ss := &binlog.SourceStatus{}`